frontend/public/actions/features.ts

```diff
@@ -183,7 +183,7 @@ const detectCanCreateProject = (dispatch) =>
       const status = _.get(err, 'response.status');
       if (status === 403) {
         dispatch(setFlag(FLAGS.CAN_CREATE_PROJECT, false));
-        dispatch(setCreateProjectMessage(err.message));
+        dispatch(setCreateProjectMessage(_.get(err, 'json.details.causes[0].message')));
```

Comment:
Shouldn't this be...?

```suggestion
        dispatch(setCreateProjectMessage(_.get(err, 'details.causes[0].message')));
```

Reply:
Tried that at first, but the `fetch` reject response is structured differently.